### ISSUE TITLE
✍️ Contribution period: Valentina Buoro

### ISSUE THREAD

**Issue by valentina-buoro** [ersilia-os/ersilia#981]
### Week 1 - Get to know the community

- [X] Join the communication channels
- [X] Open a GitHub issue (this one!)
- [x] Install the Ersilia Model Hub and test the simplest model
- [x] Install Docker if needed, and test another model
- [x] Write a motivation statement to work at Ersilia
- [x] Submit your first contribution to the Outreachy site

### Week 2 - Get Familiar with Machine Learning for Chemistry

- [ ] Select a model from the list suggested in GitBook
- [ ] Download and serve the model via the Ersilia Model Hub to ensure it works
- [ ] Open a repository on your GitHub user with all the necessary files
- [ ] Select and clean a dataset of 1000 molecules (example notebook 1)
- [ ] Run predictions for the molecules on the selected model and evaluate the results

### Week 3 - Validate a Model in the Wild

- [ ] Find a suitable dataset with sufficient experimental results
- [ ] Clean and standardize the dataset
- [ ] Run predictions and calculate metrics.

### Week 4 - Prepare your final application

- [ ] Submit the final application in the Outreachy website

**Comment by valentina-buoro** [comment]
Successfully tested ersilia using model eos3b5e  🎉

<img width="1044" alt="ersilia" src="https://github.com/ersilia-os/ersilia/assets/62246293/80641177-34d2-4264-b03d-dad1c5575670">

**Comment by valentina-buoro** [comment]
Successfully tested ersilia with docker. Initially, I got errors which I believe to be as a result of network issues. The tests went sucessfully after handling that.
<img width="706" alt="docker_pull" src="https://github.com/ersilia-os/ersilia/assets/62246293/0560cbfe-8e37-43d0-9865-10157216980a">

<img width="1099" alt="serving_model" src="https://github.com/ersilia-os/ersilia/assets/62246293/ae1f8692-20ff-4599-bb75-412c21f5a498">
<img width="1440" alt="final_result" src="https://github.com/ersilia-os/ersilia/assets/62246293/7194f74e-f8ed-42c9-8e77-3d94d9ad7f90">

**Comment by valentina-buoro** [comment]
Hello everyone, my name is Valentina Buoro, an outreachy 2024 applicant. I joined outreachy because of the opportunity that it presented in breaking barriers that I faced as a self-taught developer.  I'd love to contribute to Ersilia because I believe my background and interests closely align with the  goals of the project. I studied pharmacy at the university, but I've gone on to become a self-taught full time software developer because of my love for technology, as such contributing to Ersilia gives me the opportunity to combine my love for both healthcare and technology. I have independently learnt Python and have become proficient in it, especially as relates to data science. Also, I and some friends recently participated and won a hackathon in Artificial Intelligence for Healthcare organized by Autogon AI. In addition to that I am very eager and determined to learn all the information and skills that will be necessary for my successful contribution to Ersilia.

Working with Ersilia presents me with the opportunity to learn from, and work with experts in Artificial Intelligence and Machine learning, and I can’t state how exciting that is for me. This is why during the period of internship, I plan to work closely with my mentor, giving reports as frequently as required, participate in standups, and even pair-programming as often as I need to. 

After the internship, I plan to continue pursuing a career in AI/ML, especially in healthcare. I also plan to continue to actively participate in Ersilia’s community, especially helping new contributors like me to AI/ML.
